lib/garage_client/railtie.rb

```diff
@@ -0,0 +1,17 @@
+module GarageClient
+  class Railtie < ::Rails::Railtie
+    initializer :garage_client do |app|
+      RailsInitializer.set_default_name
+    end
```

Comment:
Is it sure this initializer will be executed **after** user defined initializer (like `config/initializers/garage_client.rb`)? If not, `GarageClient.configuration.options[:name]` in L10 will be always `nil`, I think.

Or, if we tolerate overwritting, this initializer can be executed before user defined initalizer. In that case, the "name option" will be overwritten by user defined initalizer.

In my opinion, we can tolerate overwritting. So +1 for this implementation.